### ISSUE TITLE
Better error message when your kerberos credentials are expired...

### ISSUE THREAD

**Issue by marcmengel** [marcmengel/jobsub_lite#310]
Currently you get:
```
Traceback (most recent call last):
  File "/home/mengel/jobsub_lite/bin/jobsub_submit", line 318, in <module>
    main()
  File "/home/mengel/jobsub_lite/bin/jobsub_submit", line 215, in main
    proxy, token = get_creds(varg)
  File "/home/mengel/jobsub_lite/lib/creds.py", line 31, in get_creds
    p = fake_ifdh.getProxy(role, args.get("verbose", 0), args.get("force_proxy", False))
  File "/home/mengel/jobsub_lite/lib/fake_ifdh.py", line 207, in getProxy
    "Cigetcert failed to get proxy due to kerberos issue.  Please ensure "
Exception: Cigetcert failed to get proxy due to kerberos issue.  Please ensure you have valid kerberos credentials.
```
The cigetcert message at the bottom is okay, but we should lose the stack trace.   Maybe just change it to raising SystemExit?

**Comment by marcmengel** [comment]
In fact, maybe we should change ALL raise statements to SystemExit unless we're really really sure we catch the exception somewhere?

**Comment by marcmengel** [comment]
Closed with #334
